recipes/py6s/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "py6s" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "e16cc28e54a59d3908b3902744d261046630e6811cb50b6a13e746f68d421b8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/1c/ea/fdb4fe3e289e3c14e3bdc473f6b6f595ceb476399600fab21c92db97c82a/Py6S-1.7.0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pysolar ==0.6
```

Comment:
You can remove all the `run` time dependencies from the `build` section. Leave only `python` and `setuptools`.